### PR TITLE
ci: run tests daily at 5:17 UTC (israel morning)

### PR DESCRIPTION
**Pull request by blumamir**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: Build
 on:
   pull_request:
     branches: [master]
+    
+  schedule:
```
Comment (nirsky) [inline .github/workflows/test.yml:7]: doesn't the script have since master?
That means it won't on any package, no?

Comment (blumamir) [reply .github/workflows/test.yml:7]: I don't see it in the file. Am I missing it maybe?

Comment (nirsky) [reply .github/workflows/test.yml:7]: it's in the main package.json script:
`"test:ci": "lerna run test:ci --since origin/master",`

Comment (nirsky) [reply .github/workflows/test.yml:7]: the reason is when you change a package in a pr, only the relevant tests will run

Comment (blumamir) [reply .github/workflows/test.yml:7]: Good catch, thanks. Will update it

Comment (nirsky) [reply .github/workflows/test.yml:7]: Also what about the slack notification? I think I'd create a different file for this task..
We also don't need the prettier to run daily

File: .github/workflows/daily-test.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Run Tests Daily
+
+on:
+  schedule:
+    - cron:  '17 5 * * *'
```
Comment (nirsky) [inline .github/workflows/daily-test.yml:5]: maybe add a comment that this is 5:17 UTC
i always forget how to read cron and it's nice to have a reminder

Comment (blumamir) [reply .github/workflows/daily-test.yml:6]: good idea. added

**Review by nirsky — APPROVED** [review]

**Comment by blumamir** [comment]
https://aspecto-io.monday.com/boards/312932205/pulses/1157217343

**Pull request merged**